packages/document/docs/en/config/html/tags.mdx

```diff
@@ -16,9 +16,13 @@ type HtmlTag = {
   hash?: boolean | string | ((url: string, hash: string) => string);
   /** @default true */
   publicPath?: boolean | string | ((url: string, publicPath: string) => string);
-  /** @default true */
+  /**
+   * Add this tag to the end / beginning of the original tags
```

Comment:
```suggestion
   * Sets the insertion position of the current tag relative to the original tags.
   * If set to `true` it will be inserted after the original tags, if set to `false` it will be inserted before the original tags.
```